src/main/java/org/apache/commons/imaging/icc/IccTag.java

```diff
@@ -73,11 +76,12 @@ private IccTagDataType getIccTagDataType(final int quad) {
     }
 
     public void dump(final String prefix) throws ImageReadException, IOException {
-        final PrintWriter pw = new PrintWriter(new OutputStreamWriter(System.out, Charset.defaultCharset()));
-
-        dump(pw, prefix);
-
-        pw.flush();
+        try (StringWriter sw = new StringWriter(); PrintWriter pw = new PrintWriter(sw)) {
```

Comment:
Same comment as above even though this code is slightly different - no catch clause.

Reply:
This was intentional.

The previous code contains the `throws IOException` (while the other methods don't) -> https://github.com/apache/commons-imaging/blob/d2ec76bd10f30c39ae5180ede1254908e76045f0/src/main/java/org/apache/commons/imaging/icc/IccTag.java#L75

Should we make them all the same, catching and logging, instead of throwing the IOException? As this is a `dump` method, not used by another method such as `toString()`, I think the impact on end users should be minimal.